packages/block-editor/src/components/block-list/block.js

```diff
@@ -87,7 +87,7 @@ function BlockListBlock( {
 		isDraggingBlocks && ( isSelected || isPartOfMultiSelection );
 
 	// Determine whether the block has props to apply to the wrapper.
-	if ( ! lightBlockWrapper && blockType.getEditWrapperProps ) {
+	if ( blockType.getEditWrapperProps ) {
```

Comment:
I'm not satisfied about how I extended the new block wrapper using `getEditWraperProps`. It does the job though. We'd need to figure out the best filter/API to add props to the block wrapper.

I was wondering whether we should just use the already existing getSaveProps filter https://github.com/WordPress/gutenberg/pull/20721#discussion_r390373229 if we use `Block.*` wrapper in save as well

Reply:
Noting that Block.div and similar in the edit method is a bit concerning knowing that it needs to be supported in React Native world. I might be missing the full context. 

I’m also aware that getEditWraperProps isn’t any better in that regard 😅